src/main/java/com/spacesongr/songr/models/Models.java

```diff
@@ -0,0 +1,4 @@
+package com.spacesongr.songr;
```

Comment:
double check this package name - see your AlbumRepository for an example of how to do it